### ISSUE TITLE
The deofuscator appears to break when emojis are involved

### ISSUE THREAD

**Issue by komen205** [giacomoferretti/paranoid-deobfuscator#7]
I've been having problems deobfuscating applications with obfusctaed emojis.

```        java.lang.StringBuilder sb = new java.lang.StringBuilder();
        sb.append("qweq4234weqwe");
        sb.append("qweqw234eqwe");
        sb.append("qwe \uD83D\uDE05 qweqwe");
``` 

Led to the error

Traceback (most recent call last):
  File "deobfuscator.py", line 304, in <module>
    main(args)
  File "deobfuscator.py", line 284, in main
    f.writelines(data)
  File line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 22-34: character maps to <undefined>

**Comment by giacomoferretti** [comment]
Haven't tested the emojis with the Python deobfuscation method.

**Comment by giacomoferretti** [comment]
Fixed in 47443813049710df6bf69d63fecec8e58a0f8554
